presto-docs/src/main/sphinx/release/release-0.211.rst

```diff
@@ -0,0 +1,51 @@
+=============
+Release 0.211
+=============
+
+General Changes
+---------------
+
+* Fix missing final query plan in ``QueryCompletedEvent``. Statistics and cost estimates
+  are removed from the plan text because they may not be available during event generation.
+* Update the default value of the ``http-server.https.excluded-cipher`` config
+  property to exclude non FS cipher suites. Specifically, all
+  ciphers without forward secrecy, and all ciphers that use the RSA key exchange
```

Comment:
Great explanation. This is much better than what I came up with. I think we can shorten this a bit:

> Specifically, all ciphers without forward secrecy or that use the RSA key exchange are excluded by default.